pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "Gnumeric-py"
+dynamic = ["version", "license", "authors", "maintainers"]
+description="A python library for reading and writing Gnumeric files."
+readme = "README.md"
+# maybe require newer python for build system?
+requires-python = ">=3.3.0"
+classifiers=['Development Status :: 3 - Alpha',
+            'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
```

Comment:
I've been considering switching the license to Apache 2.  Since you're contributing code, any objections to using the Apache 2 license for this PR and [your other](https://github.com/lipschultz/gnumeric-py/pull/2)?